### PR TITLE
Implemented decoding metrics

### PR DESCRIPTION
**Pull request by piosz**

### CODE REVIEW

File: core/metrics.go [hunk 1]
```diff
@@ -0,0 +1,318 @@
...
+import (
+	"time"
+
+	source_api "k8s.io/heapster/sources/api"
+)
+
+// Stub out for testing
+var timeSince = time.Since
+
+var SupportedMetrics = []SupportedMetric{
```
Comment (mwielgus) [inline core/metrics.go]: We will support all metrics, not only these mentioned below. Name it as StandardMetrics or so.

File: core/metrics.go [hunk 2]
```diff
@@ -0,0 +1,318 @@
...
+package core
+
+import (
+	"time"
+
+	source_api "k8s.io/heapster/sources/api"
+)
+
+// Stub out for testing
+var timeSince = time.Since
```
Comment (mwielgus) [inline core/metrics.go]: Is it still used?

Comment (piosz) [inline core/metrics.go]: Done

File: core/metrics.go (hunk 1 above)
Comment (piosz) [inline core/metrics.go]: Done

**Comment by vishh** [comment]
Cadvisor exposes docker image labels. Make sure we preserve that.

**Comment by piosz** [comment]
I added TODO to export all supported labels.

**Comment by piosz** [comment]
Oh, I can see Image field. I missed it previously. Will add in follow-up PR.

**Pull request merged**